cache/cacheCleaner.go

```diff
@@ -9,44 +9,41 @@ import (
 	"github.com/boltdb/bolt"
 )
 
-type cacheController struct {
-	db          *bolt.DB
+type CacheCleaner struct {
 	dbFile      *os.File
 	maxFileSize int64
 	fillFactor  float64
 	frequency   *time.Ticker
 }
 
-//goland:noinspection GoExportedFuncWithUnexportedType
-func New(db *bolt.DB, dbFile *os.File, maxFileSize int64, fillFactor float64, frequency *time.Ticker) *cacheController {
-	return &cacheController{
-		db:          db,
+func NewCacheCleaner(dbFile *os.File, maxFileSize int64, fillFactor float64, frequency *time.Ticker) *CacheCleaner {
+	return &CacheCleaner{
 		dbFile:      dbFile,
 		maxFileSize: maxFileSize,
 		fillFactor:  fillFactor,
 		frequency:   frequency,
 	}
 }
 
-func (c *cacheController) Observe() {
+func (props *CachingProperties) Observe() {
 	for {
-		<-c.frequency.C
-		if c.isSizeExceeded() {
-			c.deleteExpiredCache()
+		<-props.cleaner.frequency.C
+		if props.isSizeExceeded() {
+			props.deleteExpiredCache()
 		}
 	}
 }
 
-func (c *cacheController) isSizeExceeded() bool {
-	fileInfo, err := c.dbFile.Stat()
+func (props *CachingProperties) isSizeExceeded() bool {
+	fileInfo, err := props.cleaner.dbFile.Stat()
 	if err != nil {
-		log.Printf("Error getting file info in cacheController: %v", err) //todo: how to handle this error properly?
+		log.Printf("Error getting file info in CacheCleaner: %v", err) //todo: how to handle this error properly?
 	}
 
-	return float64(fileInfo.Size()) > float64(c.maxFileSize)*c.fillFactor
+	return float64(fileInfo.Size()) > float64(props.cleaner.maxFileSize)*props.cleaner.fillFactor
 }
 
-func (c *cacheController) deleteExpiredCache() {
+func (props *CachingProperties) deleteExpiredCache() {
```

Comment:
https://google.github.io/styleguide/go/decisions#single-letter-variable-names

For a [method receiver variable](https://google.github.io/styleguide/go/decisions#receiver-names), a one-letter or two-letter name is preferred.

Reply:
👌